### DISCUSSION TITLE
How to get HTML text encoding

### DISCUSSION THREAD

**Discussion by ghost** [curl/curl#12637] [Q&A] [answered]
How to get Content-Type: text/html; charset=utf-8 in HTTP response header  

utf8 encoding type, the current LIBC URL, whether you can use curl_easy_getinfo to get the corresponding HTML text encoding

**Comment by jay** [comment, marked as answer]
Refer to [CURLINFO_CONTENT_TYPE](https://curl.se/libcurl/c/CURLINFO_CONTENT_TYPE.html) or [curl_easy_header](https://curl.se/libcurl/c/curl_easy_header.html) doc, both have example sections that show how to obtain Content-Type.
